Common/SimConfig/src/SimConfig.cxx

```diff
@@ -44,6 +44,7 @@ void SimConfig::initOptions(boost::program_options::options_description& options
     "chunkSizeI", bpo::value<int>()->default_value(-1), "internalChunkSize")(
     "seed", bpo::value<int>()->default_value(-1), "initial seed (default: -1 random)")(
     "field", bpo::value<int>()->default_value(-5), "L3 field rounded to kGauss, allowed values +-2,+-5 and 0")(
+    "uniformField", bpo::value<bool>()->default_value(false), "set uniform magnetic field")(
```

Comment:
In principle a nice feature! However, I would slightly prefer not to add more options (but rather use existing ones or use the ConfigParams mechanism) though. Couldn't we here just use the existing "field" option but extend it with some "type" information. Such as "--field -5U" where U can be interpreted as "uniform" and if not given ... we'd take the usual ALICE field.

Reply:
As it stands the int value of the field is passed to construct the magnetic field. The alternative is to replace the field type from int to string and parse when needed. Say, replace the argument of [MagneticField* MagneticField::createNominalField(std::string fld)](https://github.com/AliceO2Group/AliceO2/blob/0f327ec83dfa44c69eaaf33cb18cdd8b7d83a53e/Common/Field/src/MagneticField.cxx#L151) ?
On the GRP this is not possible, since the magnet currents are stored explictly. GRP should remain as it in the current PR. That's what you are suggesting? @sawenzel

Reply:
I realize parsing can be done right after reading the arguments while filling SimConfig.

Reply:
@rpezzi yes, in the GRP the flag of uniform field must stay separated. For the "-5U" option, yes, you can pass to the createNominalField a string and parse it there, but you will need to parse it also in o2sim.C, when creating a GRP object.